README.md

```diff
@@ -27,6 +27,8 @@ To work on umd.io, or to run your own instance, start by forking and cloning thi
    - You might need to run docker-related commands with `sudo` if you're a linux user
 4. Run the scrapers `./umdio.sh scrape`
    - You might need to `chmod +x umdio.sh`
+5. Clone the umdio-data submodule
+   - `cd data && git submodule init && git submodule update`
```

Comment:
just `git submodule update --init` is probably cleaner?

Reply:
Nice catch - goes to show my git submodules knowledge, haha

Reply:
oh you don't even need the `cd data`, should have clarified that earlier